zebra-consensus/src/checkpoint.rs

```diff
@@ -217,7 +219,7 @@ mod tests {
     #[tokio::test]
     #[spandoc::spandoc]
     async fn checkpoint_not_present_fail() -> Result<(), Report> {
-        install_tracing();
+        zebra_test::init();
```

Comment:
I've already committed the `zebra_test::init()` changes to main.

Reply:
aah, cool. I'll rebase way before I merge this stuff,